doc/api/cluster.md

```diff
@@ -626,6 +626,8 @@ values are `"rr"` and `"none"`.
     (Default=`process.argv.slice(2)`)
   * `silent` {Boolean} whether or not to send output to parent's stdio.
     (Default=`false`)
+  * `stdio` {Array} Configures the stdio of forked processes. When this option
+    is provided, it overrides `silent`.
```

Comment:
This should explain that fd 3 _must_ be 'ipc', otherwise the cluster module won't work.  It's unfortunate in that it leaks what is essentially an implementation detail.